app/controllers/HomeController.scala

```diff
@@ -84,15 +87,17 @@ class HomeController @Inject()(assets: Assets,configuration: Configuration, cc:
               }
             case resp@HttpResponse(code, _, _, _) => {
               resp.discardEntityBytes()
-              Future("")
+              //Future("")
+              val objectError:Future[String]=Future.failed(throw new IOException(code.defaultMessage()))
```

Comment:
You could condense this into a single line by just using 
`Future.failed(throw new IOException(code.defaultMessage()))`

Reply:
Necessary Changes Done Successfully....